litmus-portal/backend/subscriber/pkg/gql/subscriptions.go

```diff
@@ -80,7 +83,7 @@ func ClusterConnect(clusterData map[string]string) {
 			// send pod logs
 			logrus.Print("LOG REQUEST ", podRequest)
 			SendPodLogs(clusterData, podRequest)
-		} else {
+		} else if strings.ToLower(r.Payload.Data.ClusterConnect.Action.RequestType) == "create" {
```

Comment:
RequestType can be `update`, `delete`, `get`. We should not put one type, this type will be decided by clusteroperation function

Reply:
This needs a change in the workflow query because I kept the requesttype as the type of action that's needed. I can make it like "update" || "create" || "delete"... That should also solve it.

Reply:
We don't have to make another check, it's already there in clusteroperations

Reply:
Updated it, now it should work for all the 4 types.